encord/storage.py

```diff
@@ -1091,6 +1096,27 @@ def dicom_series_uid(self) -> Optional[str]:
     def frame_count(self) -> Optional[int]:
         return self._orm_item.frame_count
 
+    def validate_metadata(self) -> bool:
+        """Check if the metadata for this storage item validates against the metadata schema
+
+        Returns True if validates correctly. Throws EncordException indicating which field failed to validate
```

Comment:
This should raise some very specific exception type (which should be a *subclass* of EncordException). Otherwise it would be annoying to try to figure out what went wrong: validation or the update itself.

Reply:
Done!